### ISSUE TITLE
Cannot see Public Questions Previously Asked and Answered

### ISSUE THREAD

**Issue by hansontranhai** [Plaxy-Technologies-Inc/YouPlanets-Bug-Report#62]
I'm not sure if it was intended that way. But if it's a public question, shouldn't it be available publicly? At least to members who have logged in?

![image](https://user-images.githubusercontent.com/77167655/104503879-475b5780-55a7-11eb-8fab-332d1cd1900d.png)

**Comment by JalenYes** [comment]
You are right. This is one of the technical debt. We should build it to be the same as the design below. Let's do it in YP V1.3. @AdvaithNair 
![image](https://user-images.githubusercontent.com/40333215/104504326-4975e400-55b0-11eb-9b8d-6d5afbbd2590.png)
